src/herbpy/herbrobot.py

```diff
@@ -56,7 +86,54 @@ def __init__(self, left_arm_sim, right_arm_sim, right_ft_sim,
         accel_limits[self.left_arm.GetArmIndices()] = [ 2. ] * self.left_arm.GetArmDOF()
         accel_limits[self.right_arm.GetArmIndices()] = [ 2. ] * self.right_arm.GetArmDOF()
         self.SetDOFAccelerationLimits(accel_limits)
-        
+
+
+        # Determine always-on controllers
+
+        # hand controllers
+        if not left_hand_sim:
+            self.controllers_always_on.append('left_hand_controller')
+
+        if not right_hand_sim:
+            self.controllers_always_on.append('right_hand_controller')
+
+        # force/torque controllers
+        if not left_ft_sim or not right_ft_sim:
+            self.controllers_always_on.append('force_torque_sensor_controller')
```

Comment:
`force_torque_sensor_controller` loads a single controller that publishes all available F/T resources. So we get the appropriate message in each individual hand's call.